### PR TITLE
chore: add google analytics for user metrics collection

### PR DESCRIPTION
**Pull request by saladgg**

### CODE REVIEW

File: assets/static/js/project.js [hunk 1]
```diff
@@ -33,37 +33,3 @@ function switchDashboards(event) {
     // Hide the menu after selection
     $("#accordionSidebar.toggled #dashboards-menu").removeClass("show");
 }
-
-
-(function($) {
```
Comment (kennedykori) [inline assets/static/js/project.js:38]: Why did you delete this 😕??

File: assets/templates/atoms/head.html [hunk 2]
```diff
@@ -35,7 +35,20 @@
...
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-WW2W29ZMTZ"></script>
+
     <script src="https://kit.fontawesome.com/e1c9d8c99d.js" crossorigin="anonymous"></script>
     {% endblock javascript %}
+
+
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-S5JLWM36RQ"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
```
Comment (kennedykori) [inline assets/templates/atoms/head.html:46]: I think we should strive to keep very little `javascript` embeded in `html` if not remove it entirely. This way, it easier to manage the `javascript` and might make it easier for us to switch to `Typescript` in the future.
I would recommend you move this to the `project.js` file.

**Review by kennedykori — CHANGES_REQUESTED** [review]

File: assets/static/js/project.js (hunk 1 above)
Comment (saladgg) [reply assets/static/js/project.js:38]: It's the function in which google analytics script was embedded. Its actions seems not applicable in our case.  With/without this function, we can show/hide dashboards, minimize and maximize the sidebar. I don't see where it applies unless you found something.

File: assets/templates/atoms/head.html (hunk 2 above)
Comment (saladgg) [reply assets/templates/atoms/head.html:46]: Agreed.  I'll move it to `project.js` as suggested

**Review by kennedykori — APPROVED** [review]
